src/Views/Form/Templates/Sequoia/Sequoia.php

```diff
@@ -65,47 +65,53 @@ public function loadScripts() {
 		$templateOptions['payment_information']['header_label']   = ! empty( $templateOptions['payment_information']['header_label'] ) ? $templateOptions['payment_information']['header_label'] : __( 'Add Your Information', 'give' );
 		$templateOptions['payment_information']['checkout_label'] = ! empty( $templateOptions['payment_information']['checkout_label'] ) ? $templateOptions['payment_information']['checkout_label'] : __( 'Process Donation', 'give' );
 
-		wp_enqueue_style( 'give-google-font-montserrat', 'https://fonts.googleapis.com/css?family=Montserrat:100,100i,200,200i,300,300i,400,400i,500,500i,600,600i,700,700i,800,800i,900,900i&display=swap', [], GIVE_VERSION );
-		wp_enqueue_style( 'give-sequoia-template-css', GIVE_PLUGIN_URL . 'assets/dist/css/give-sequoia-template.css', [ 'give-styles' ], GIVE_VERSION );
+		wp_enqueue_style( 'give-google-font-montserrat', 'https://fonts.googleapis.com/css?family=Montserrat:100,100i,200,200i,300,300i,400,400i,500,500i,600,600i,700,700i,800,800i,900,900i&display=swap', array(), GIVE_VERSION );
+		wp_enqueue_style( 'give-sequoia-template-css', GIVE_PLUGIN_URL . 'assets/dist/css/give-sequoia-template.css', array( 'give-styles' ), GIVE_VERSION );
 
 		$primaryColor = $templateOptions['introduction']['primary_color'];
-		$dynamic_css  = sprintf(
-			'
+		$rawColor     = trim( $primaryColor, '#' );
+		$dynamic_css  = "
```

Comment:
@henryholtgeerts I saw that you remove `sprintf` in favor of `$rawColor`, but I will suggest that create a new variable for addon related styling and keep `sprintf` that way it will be maintainable. A new variable for non-core related styling will help to create a separation between core and addon style.

Reply:
I went forward with creating a recurring dynamic css variable, but opted to use string interpolation because, in this instance it is more readable (sprintf would require escaping a number of % characters in the checkbox background rule). See: https://github.com/impress-org/givewp/pull/4711/commits/3d62979f42e84457aaaab29b90cd2f2e24dc95e3